### PR TITLE
feat: implement PUM calculation

### LINKED ISSUE #8: Implement computation of PUM

### PR DESCRIPTION
**Pull request by mantaur**
fixes #8
wip: Started work on a possible test for PUM calculation, but it's value as a test is arguably low

### CODE REVIEW

**Comment by psalqvist** [comment]
Should we refine the test before merging into main?

**Comment by mantaur** [comment]
By all means if there's value in it. As I was working on the test I kinda realised how unneccessarily complex and intricate the test became for how simple the functionality of the PUM calculation was. I therefor named the file .wip, so it's not actually used, so I'm vying for not using a test for the PUM at all (at least not one where the test is so hard to maintain and understand it's actually easier to just go and look at the PUM code to verify/assert that it does what it's supposed to do).

**Comment by nolanderc** [comment]
Essentially, the PUM is just computing a bunch of binary operators. Instead of testing a full 15x15 matrix, we can probably get away with only testing the 3 operators.

The primary issue in the PUM calculation (modulo any issues with the operators) is the indexing into the CMV. This is probably best done with an integration test later on.

File: src/main/java/decide/ConditionsMetVector.java [hunk 1]
```diff
@@ -11,8 +11,20 @@
...
+        this.conditions[5] = LIC5(input);
+        this.conditions[6] = LIC6(input);
+        this.conditions[7] = LIC7(input);
+        this.conditions[8] = LIC8(input);
+        this.conditions[9] = LIC9(input);
+        this.conditions[10] = LIC10(input);
+        this.conditions[11] = LIC11(input);
+        this.conditions[12] = LIC12(input);
+        this.conditions[13] = LIC13(input);
+        // this.conditions[14] = LIC14(input);
```
Comment (nolanderc) [inline src/main/java/decide/ConditionsMetVector.java]: These should preferably be moved to their own PR/commit.

**Review by nolanderc — CHANGES_REQUESTED** [review]
Move CMV-related parts into its own PR/commit

**Review by nolanderc — APPROVED** [review]
